samples/sample_common/src/vaapi_utils_drm.cpp

```diff
@@ -343,7 +344,7 @@ bool drmRenderer::getPlane()
         if (plane) {
             if (plane->possible_crtcs & (1 << m_crtcIndex)) {
                 for (uint32_t j = 0; j < plane->count_formats; ++j) {
-                    if (plane->formats[j] == DRM_FORMAT_XRGB8888) {
+                    if (plane->formats[j] == DRM_FORMAT_NV12) {
```

Comment:
@VUNNYSODHI , can you please explain more about this place? E.g. will RGB continue working after the change?

Reply:
We just want to test the NV12 mode, if RGB still needs to be continued working , we can change it as below:

 if ((plane->formats[j] == DRM_FORMAT_XRGB8888) || (plane->formats[j] == DRM_FORMAT_NV12)) {

Reply:
Yes, please make sure RGB continues working.

Reply:
I have submitted changes, tested  RGB and NV12 both are working fine.